.github/workflows/foundation-ang-trigger.yml

```diff
@@ -0,0 +1,56 @@
+# Controls when the action will run. The workflow runs when manually triggered using the UI or API
+
+name: Foundation_Ang_On_Demand
+
+on:
+  push:
+    branches:
+      - dev/angular13  
+  workflow_dispatch:
+    # Inputs the workflow accepts.
+    inputs:
+      foundation_ang_version:
+        description: 'Foundation-ang version'
+        required: true
+        default: '7.0.0'
+
+# A workflow run is made up of one or more jobs that can run sequentially or in parallel
+jobs:
+  build_web:
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: Checkout the code
+        uses: actions/checkout@v3
+
+      - name: Derive appropriate SHAs for base and head for `nx affected` commands
+        uses: nrwl/nx-set-shas@v2
+        with:
+          main-branch-name: ${{ github.base_ref }}
+
+      - name: Setup Node.js
+        uses: actions/setup-node@v3
+        with:
+          node-version: 16
+          cache: 'npm'
+          registry-url: 'https://repo.backbase.com/artifactory/api/npm/npm-backbase/'
+          scope: '@backbase'
+
+      - name: Install Dependencies
+        run: npm ci --legacy-peer-deps
+        env:
+          NODE_AUTH_TOKEN: ${{ secrets.NPM_TOKEN }}
+
+        if: ${{ github.event_name == push && github.ref_name == dev/angular13 }}
```

Comment:
Why do we need this condition?

Reply:
We have two separate events in the `foundation-ang-trigger.yml`:

- push
- workflow_dispatch

In both events, I overwrote the foundation-ang version.  The first event uses the environment variable and the second event uses the user's input which is the same as the environment variable (7.0.0) but can be different based on the user input. So, based on the event name, I have the different conditions to overwrite foundation-ang suitable version.